### PR TITLE
"Update fields" feature

### PR DESCRIPTION
**Pull request by Borales**
For example, to be able to update other fields of the domain record (like "port", "priority" etc).

### CODE REVIEW

**Comment by Borales** [comment]
@toin0u just to clarify - is this repo still maintained? or should I work in a separate fork?

**Comment by GrahamCampbell** [comment]
Yeh, it's still maintained. :)

**Comment by GrahamCampbell** [comment]
Is this totally BC?

File: src/Api/DomainRecord.php [hunk 1]
```diff
@@ -135,8 +129,22 @@ public function update($domainName, $recordId, $name)
...
+    /**
+     * @param string $domainName
+     * @param int    $recordId
+     * @param array  $fields
+     *
+     * @throws \RuntimeException
+     *
+     * @return DomainRecordEntity
+     */
+    public function updateFields($domainName, $recordId, $fields)
```
Comment (GrahamCampbell) [inline src/Api/DomainRecord.php]: Could you add test coverage for this method please.

**Comment by Borales** [comment]
@GrahamCampbell 
> Is this totally BC?
It's totally BC! `update` and `updateData` use the same code to make a PUT request.
I just moved that code to a single `updateFields` method, where you can set any fields to be updated.

**Comment by Borales** [comment]
@GrahamCampbell sorry, not quite familiar with the PhpSpec methodology, but I've tried to add the `updateFields` test as you asked.

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Comment by Borales** [comment]
@GrahamCampbell could you please update the composer/packagist data as well?

**Pull request merged**